logviewer/src/main/java/org/togetherjava/logwatcher/watcher/StreamWatcher.java

```diff
@@ -0,0 +1,52 @@
+package org.togetherjava.logwatcher.watcher;
+
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class StreamWatcher {
+    private static final Map<UUID, Runnable> consumerMap = new ConcurrentHashMap<>(3);
```

Comment:
Why `3`? Use a constant with a good name instead.

Reply:
3 is just to have an small initial Size, since I don't think more than 3 people will watch the logs at once

Reply:
Fair. `EXPECTED_CONCURRENT_LOG_WATCHERS = 3` might be helpful then.